src/commons/utils/date.ts

```diff
@@ -0,0 +1,26 @@
+import { EventDuration } from "@/apollo/types";
+import { format } from "date-fns";
+
+export const getDisplayDate = (durations?: EventDuration[]) => {
+  if (!durations) {
+    return "-";
+  }
+  const startDateText = format(new Date(durations[0].start), "ccc, d MMM");
+  if (durations.length !== 0) {
+    const endDateText = format(
+      new Date(durations[durations.length - 1].start),
+      "d MMM"
+    );
+    return `${startDateText} - ${endDateText}`;
+  }
+  return startDateText;
+};
+
+export const getMainTimetable = (durations?: EventDuration[]) => {
+  if (!durations || durations.length !== 0) {
+    return "-";
+  }
```

Comment:
Then the only code that will fail this test and go to next lines is blank durations list []. I do not think this is what you want.

Reply:
That is what I want.

Reply:
I have consulted with UXUI krub